src/redux/books/Slice-Book.js

```diff
@@ -0,0 +1,40 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [
+  {
+    id: 'Book-1',
+    title: 'To Kill a Mockingbird,',
+    author: 'Harper Lee',
+    category: 'Fiction',
+  },
+  {
+    id: 'Book-2',
+    title: '1984',
+    author: 'George Orwell',
+    category: 'Fiction',
+  },
+  {
+    id: 'Book-3',
+    title: 'The Lord of the Rings',
+    author: ' J.R.R. Tolkien',
+    category: 'Fiction',
+  },
+];
+
+const bookSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    bookAdd: (state, action) => {
+      state.push(action.payload);
+    },
```

Comment:
- [OPTIONAL] I highly recommend not using the `push` because it mutates the array, You can use `Destructuring assignment` which will not mutate the array.

  - Suggestion: 
    `bookAdd: (state, action) => {
      state = [...state, action.payload];
      return state;
    }`

Reply:
Oh ok...nice..it a great suggestion thanks a lot.